com.unity.shadergraph/Editor/GraphUI/EditorCommon/CommandStateObserver/ShaderGraphCommandOverrides.cs

```diff
@@ -258,13 +258,13 @@ RenameElementCommand renameElementCommand
                 var nodeWriter = shaderGraphModel.GraphHandler.GetNodeWriter(nodeModel.graphDataName);
                 if (nodeWriter != null)
                 {
-                    var vec4Value = updatePortConstantCommand.NewValue is Vector4 value ? value : default;
-                    nodeWriter.SetPortField(portModel.UniqueName, "c0", vec4Value.x);
-                    nodeWriter.SetPortField(portModel.UniqueName, "c1", vec4Value.y);
-                    nodeWriter.SetPortField(portModel.UniqueName, "c2", vec4Value.z);
-                    nodeWriter.SetPortField(portModel.UniqueName, "c3", vec4Value.w);
+                    // var vec4Value = updatePortConstantCommand.NewValue is Vector4 value ? value : default;
+                    //nodeWriter.SetPortField(portModel.UniqueName, "c0", vec4Value.x);
+                    //nodeWriter.SetPortField(portModel.UniqueName, "c1", vec4Value.y);
+                    //nodeWriter.SetPortField(portModel.UniqueName, "c2", vec4Value.z);
+                    //nodeWriter.SetPortField(portModel.UniqueName, "c3", vec4Value.w);
```

Comment:
`DefaultCommandHandler` is ALSO getting called. This would result in this getting called second and clobbering non `vec4` values with zeroes. We'll need to explore how to prevent this sort of thing in the future. I'm inclined to believe that both commands here are valid-- one should just update the local property of a preview, the other should update the value.